src/lib.rs

```diff
@@ -221,27 +150,22 @@ impl fmt::Display for ServerId {
 }
 
 /// The ID of a Raft client.
-#[derive(Copy, Clone, Hash, PartialEq, Eq)]
+#[derive(Copy, Clone, Debug, Hash, PartialEq, Eq)]
 pub struct ClientId(Uuid);
+
 impl ClientId {
-    fn new() -> ClientId {
```

Comment:
Shouldn't be. Seemed unused other than tests - Anyone implementing this would use it - however, they would need this impl to be public (which it isn't).

Making this impl pub will also remove the compile time warning that it is unused.